src/desktop/apps/auction/components/layout/auction_info/Registration.js

```diff
@@ -29,37 +30,57 @@ function Registration(props) {
         } else if (!isQualifiedForBidding) {
           return (
             <div className={b("wrapper")}>
-              <button className="avant-garde-button-black is-block is-disabled">
+              <Button width="100%" size="large" isDisabled>
                 Registration pending
-              </button>
-              <div className={b("small", { warning: true })}>
+              </Button>
+              <Sans mt="1" size="3" color="yellow100" textAlign="center">
                 Reviewing submitted information
-              </div>
+              </Sans>
             </div>
           )
         } else if (numBidders > 0) {
           return (
             <div className={b("approved")}>
               <span className="icon-check" />
-              Approved to Bid
+              <Sans mt="1" size="3" color="black60" textAlign="center">
+                Approved to Bid
+              </Sans>
             </div>
           )
         } else if (isRegistrationEnded) {
           return (
             <div className={b("wrapper")}>
-              <button className="avant-garde-button-black is-block is-disabled">
+              <Button width="100%" size="large" isDisabled>
                 Registration closed
-              </button>
-              <div className={b("small")}>Registration required to bid</div>
+              </Button>
+              <Sans mt="1" size="3" color="black60" textAlign="center">
+                Registration required to bid
+              </Sans>
             </div>
           )
         } else {
           return (
             <div className={b("wrapper")}>
-              <button className="avant-garde-button-black is-block js-register-button">
-                Register to bid
-              </button>
-              <div className={b("small")}>Registration required to bid</div>
+              <div className="js-register-button">
+                <Button width="100%" size="large">
+                  Register to bid
+                </Button>
+              </div>
+              {userNeedsIdentityVerification ? (
+                <Sans mt="1" size="3" color="black60" textAlign="center">
+                  Identity verification required to bid.{" "}
+                  <a
+                    target="_blank"
```

Comment:
The user stays within Artsy and the button is present directly on the page, not on a modal. So I think we should keep the user on the same tab/window.

```suggestion
```

Reply:
the ticket says "FAQ link opens in new tab to IDV FAQ" so I did it this way.

Reply:
Ok, this is minor and definitely not a blocker so it should be fine for now.